src/nrtk_explorer/app/core.py

```diff
@@ -50,7 +52,7 @@ def __init__(self, server=None):
 
         known_args, _ = self.server.cli.parse_known_args()
         self.input_paths = known_args.dataset
-        self.state.current_dataset = str(Path(self.input_paths[0]).resolve())
```

Comment:
don't you want to have absolute path that are properly formed?

Reply:
This helped us fix some issue on Windows

Reply:
Gotcha, I put the `Path(ds).resolve()` step where the file access happens, `dataset.py`.  Alternatively, could front load the "validation" where dataset paths are ingested from command line.